docs/user/accumulo/data_management.rst

```diff
@@ -158,110 +281,3 @@ only populate features matching a CQL filter (e.g. the last month), or choose to
 data. The update is seamless, and clients can continue to query and ingest while it runs.
 
 See :ref:`add_index_command` for more details on the command line tools.
-
-
-
-
-
-Splitting the Record Index
---------------------------
-
-By default, GeoMesa assumes that feature IDs are UUIDs, and have an even distribution. If your
-feature IDs do not follow this pattern, you may define a custom table splitter for the record index.
-This will ensure that your features are spread across several different tablet servers, speeding
-up ingestion and queries.
-
-GeoMesa supplies three different table splitter options:
```

Comment:
Ah!  Did this just get straight deleted:)?

Reply:
moved?!? ...

Reply:
this diff is highly misleading ;) I just added the section on 'accumulo logical timestamps'